tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
Since this is open source, we might want to set an engine value inside package.json, since we're targeting es2017. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines